src/sonic-yang-models/tests/yang_model_tests/tests_config/syslog.json

```diff
@@ -136,17 +136,6 @@
             }
         }
     },
-    "SYSLOG_SERVER_INVALID_IPADDR_TEST" : {
```

Comment:
Why is this test removed? Is it because syslog server can be a string and hence IP validation cannot be done?

Reply:
yes